packages/validator/templates/apiSpecifications.json

```diff
@@ -12,7 +12,7 @@
     "__conditions": [
       {
         "__if": {
-          "__this": "(?<={)[^\\/{}]+(?=})"
+          "__this_name": "(?<={)[^\\/{}]+(?=})"
```

Comment:
What's this checking for?

Reply:
There can be parameters in the path, something like this for example `/path/{param1}/{param2}`, in this case `param1` and `param2` must be present in `parameters` array of the path. The regular expression is a bit more complicated because it uses lookahead and lookbehind to check for `{}`, so that `__match` in the condition gets replaced for `param1` and not `{param1}`.